### DISCUSSION TITLE
Styling based on parent state enabled for Attributify?

### DISCUSSION THREAD

**Discussion by iamhectorsosa** [unocss/unocss#2741] [Q&A] [answered]
[Styling based on parent state](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state)

Is it enabled for attributes?

```html
<div group="~">
  <span group="..." />
</div>
```

Seems missing from `AttributifyAttributes`

**Comment by serkodev** [comment]
Regarding to https://github.com/unocss/unocss/issues/925

```html
<div group>
  <span group-hover="text-red">Hover Me</span>
</div>
```

You can do this with the following UnoCSS config
```js
presets: [
  presetAttributify(),
  presetUno({ attributifyPseudo: true }),
],
```

Check this [UnoCSS Playground Demo](https://unocss.dev/play/?html=DwEwlgbgBA5gTgewK4AcB8AoKVgGcUCGAdrIqgLQAWCEApnALwBEALrQB4vly0hNoAJGvSgBZWsAD0+YpingImIA&config=JYWwDg9gTgLgBAbwFBzgEwKYDNgDsMDCEuOA5gDQpxhQYDOGMAgjDFMAEYCuMwWAnpVQ16jAJIBjYnSHVaDGAFVcESgF84WKBBBwA5FxUS6dPUiQYAHpFjpsAQy4AbeJhz4iJYKQAUyVFDO9ABccADaVKhhehJcdDA6ALSBThh65IhwUk7QoXq0aHpwagC6VCWydAAW0DCxMHSh-qj6sfFJ1bX1enkwVjCJTqRwfZYD0Pa4pBhwNQBuGFDBowN99k5mqGqyIgqN4ZFyokoqfnD2rOzcvAIACgxcaBChbFwzagCUssLyjCxsnB4fH4Pi+h124ikuDofkOqDoEnWGFCAEYAHQAJm+LQkaFweSqrDAjQA9CT6CA0dUSZsWp9ZGVPkggA&css=Q&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g).

**Reply by iamhectorsosa** [reply]
Any reason why this isn't the default?

**Reply by serkodev** [reply, marked as answer]
I am not sure. Just like anufu's [comment](https://github.com/unocss/unocss/issues/925#issuecomment-1116349398) at the bottom said so, it's possibly to set the attribute automaticity and PR is welcomed.

if my response above is helpful, please feel free to mark it as answer. 😃

**Reply by zyyv** [reply]
> Any reason why this isn't the default?

The default usage is `class="group"`
